CombinatorialGames/Surreal/Basic.lean

```diff
@@ -456,5 +456,19 @@ theorem mk_ofSets {s t : Set IGame.{u}} [Small.{u} s] [Small.{u} t] {H : Numeric
 theorem zero_def : 0 = {∅ | ∅}ˢ := by apply (mk_ofSets ..).trans; congr <;> simp
 theorem one_def : 1 = {{0} | ∅}ˢ := by apply (mk_ofSets ..).trans; congr <;> aesop
 
+instance : DenselyOrdered Surreal.{u} where
```

Comment:
This is the perfect job for `Surreal.ofSets`. I think this can get pulled out of the PR into a separate PR which expands the `ofSets` API and uses it to give a one-liner proof.